tests/FrameCameraTests.cpp

```diff
@@ -101,3 +129,32 @@ TEST_F(FrameSensorModel, OffBody4) {
    EXPECT_NEAR(groundPt.z, -14.99325304, 1e-8);
 }
 
+TEST_F(FrameSensorModel, setFocalPlane1) {
+  int precision  = 20;
+
+   csm::Isd localIsd;
+   loadIsd("data/simpleFramerISD.json",localIsd);
+   csm::ImageCoord imagePt(7.0, 7.0);
+   double ux,uy;
+   vector<double> odtx{0.0,1.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0};
+   vector<double> odty{0.0,1.0,1.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0};
+
+   localIsd.clearParams("odt_x");
+   localIsd.clearParams("odt_y");
+   for (auto & val: odtx){
+      ostringstream strval;
+      strval << setprecision(precision) << val;
+      localIsd.addParam("odt_x", strval.str());
+   }
+   for (auto & val: odty){
+      ostringstream strval;
+      strval << setprecision(precision) << val;
+      localIsd.addParam("odt_y", strval.str());
+   }
+
+   sensorModel->setFocalPlane(imagePt.samp, imagePt.line, ux, uy);
```

Comment:
The sensor model is not updated from your local ISD.  You need to modify the ISD and then generate a camera model from it. Look at how @AstroKEW78 wrote the focal length test https://github.com/USGS-Astrogeology/CSM-CameraModel/blob/master/tests/FrameCameraTests.cpp#L127